miqcli/cli/main.py

```diff
@@ -272,8 +205,26 @@ def get_command(self, ctx, name):
         return cmd
 
 
+# set context settings w/ our configuration
+config = Config({})
+
+# check lowest precedence /etc first
+if check_yaml(MIQCLI_CFG_FILE_LOC)["exists"]:
+    config.from_yaml(check_yaml(MIQCLI_CFG_FILE_LOC)["filepath"], silent=True)
+
+# next check the local path
+if check_yaml(os.getcwd())["exists"]:
+    config.from_yaml(check_yaml(os.getcwd())["filepath"], silent=True)
+
+# next check the env var
+if 'MIQ_CFG' in os.environ and os.environ['MIQ_CFG']:
```

Comment:
same case. The function from_env will check automatically. If the environment variable is not set, nothing happens. If the environment variable is set, it loads the content. If silent is set to True and the content is wrong, nothing happens. If the silent is set to False and the content is wrong, then a failure will be raised.